airflow/cli/commands/db_command.py

```diff
@@ -218,3 +218,12 @@ def export_cleaned(args):
         table_names=args.tables,
         drop_archives=args.drop_archives,
     )
+
+
+@cli_utils.action_cli(check_db=False)
+def drop_archived(args):
+    """Drops archived tables from metadata database."""
+    drop_archived_tables(
+        table_names=args.tables,
+        confirm=not args.yes,
```

Comment:
Can you rename this to something like `needs_confirm` or `manual_confirm` — because in my head using `-y` means you have confirmed it and it just reads … wrong.